### PR TITLE
Add new version notification to admin pages

### LINKED ISSUE #119: Show newVersionAvailable dialog in Admin Panel too
It is currently shown in the client app, but not the admin app.

### PR DESCRIPTION
**Pull request by torjusti**
Fixes #119

### CODE REVIEW

**Comment by OptimusCrime** [comment]
Files changed 0?

**Comment by torjusti** [comment]
Heh...

The new version notification gets triggered when a new commit is added and the server is reloaded. I had been commiting with --allow-empty and forgot to actually add the file before pushing :baby:

**Comment by OptimusCrime** [comment]
Haha, no worries. Thought I'd just let you know :)

File: client/src/components/ConcludedIssueList.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ const calculateMajority = (issue) => {
   const numTotalVotes = Object.keys(votes).length;
   const voteObjects = Object.keys(votes).map(key => votes[key]);
 
+  console.log(issue);
```
Comment (sklirg) [inline client/src/components/ConcludedIssueList.js]: Are you sure this is how to

> Add new version notification to the ACP

? 😜

**Pull request merged**